stable/restore/README.md

```diff
@@ -225,6 +232,24 @@ Verify the restore completed successfully; view the log output from the restarte
 Finished restoring /var/opt/nuodb/backup/20190619T101450 to /var/opt/nuodb/archive/nuodb/demo. Created archive with archive ID 8
 ```
 
+### Manual archive restore
+
+For complex requirements where automatic archive restore is not sufficient, `restore.manual="true"` can be set during restore chart installation.
+The NuoDB SM engine startup will block waiting for the user to perform the archives restore manually.
```

Comment:
TEs will be blocked also.

Reply:
If `database` restore type is selected, then it will block all engines startup including TEs. If the restore type is `archive`, it's expected that the database is running, and restore for a specific archive id is needed (this is mainly to support the accelerated sync use case).
I will remove the SM reference here.